source/blog/044-rfc-indented-syntax.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Request for Comments: Indented Syntax Improvements"
+author: James Stuckey Weber
+date: 2024-12-02 15:30:00 -8
+---
+
+For users of the indented syntax (`.sass`), there has been a long standing
+request for the ability to have line breaks that don't end a statement. Certain
+CSS syntaxes like `grid-template` and `@font-face` `src` declarations are long,
+and line breaks can greatly increase the readability of the styles. 
+
+There have been many suggestions, several attempts, and numerous workarounds to
+address this pain point. We are proposing several new syntax options in the
+indented syntax that add considerable flexibility and look to address this as
+part of the language.
+
+## The new syntax
+
+Instead of adding syntax that is completely new to Sass, we propose to borrow
+syntax from the SCSS format and make it available in the indented syntax. This
+reduces the learning curve, code complexity, and likelihood of future conflicts
+with new CSS features.
+
+A common way of using this syntax will be wrapping lists with line breaks in
+parentheses. This allows for a grid-template declaration on multiple lines.
+
+```sass
+.grid
+  display: grid
+  grid-template: (
+    "header" min-content
+    "main" 1fr
+  )
+}
+```
+
+These parentheses will not be included in the output.
+
+```css
+.grid {
+  display: grid;
+  grid-template: "header" min-content "main" 1fr;
+}
+```
+
+Powering this change is the underlying rule that linebreaks only end statements
+in places where statements can end. Anywhere where a statement can't end, a
+linebreak is treated as whitespace. This means that if the parser can't tell
+whether or not a linebreak is intended to end a statement, it will end the
+statement.
+
+This example demonstrates several places where line breaks will be possible.
+
+```sass
+@each $item in /* A statement can't end after the word "in" in an `@each` statement. */
+    1, 2, 3
+  .item-#{ 
+    $item /* A statement can't end inside the curly braces in an interpolation. */
+  }
+    content: $item * /* A statement can't end after a multiplication operator. */
+        10
+```
+
+
+As a counter example, here are some places in the same stylesheet where
+linebreaks could end statements, so linebreaks will cause compilation errors.
```

Comment:
I think it's worth explaining a little more explicitly that these will cause compilation errors because the *next line* can't be parsed as a valid statement.